appengine-cron-emulator/build.gradle

```diff
@@ -18,10 +18,10 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: "$project.ext.JACKSON_VERSION"
-    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: "$project.ext.JACKSON_VERSION"
-    compile group: 'com.squareup.okhttp', name: 'okhttp', version: '2.7.5'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: "$project.ext.JACKSON_VERSION"
```

Comment:
why this choice instead of api?

Reply:
As it turns out, `api` means, "I am creating a library so expose this for downstream consumers."